### DISCUSSION TITLE
Autocomplate Problems Important

### DISCUSSION THREAD

**Discussion by 1lkin13** [heroui-inc/heroui#4218] [Feedback]
1)I can't do multi select like select item with autocomplete component
2)For large data, autocomplete freezes, cannot be optimized with react window or something else

**Comment by wingkwong** [comment]
1. Autocomplete is basically an input, not select. You can use select with multiple selection.
2. We implemented [virtualization](https://canary.nextui.org/docs/components/autocomplete#virtualization) in autocomplete (currently in canary), which will be released in the next version.

**Reply by 1lkin13** [reply]
1) It's fixed, thank you.
2)I want to search in the selection, you can add Multi Selected Keys as autocomplete, I think this is very necessary and most Ui libraries don't have something like this, this can make you stand out
